### PR TITLE
blockdev_stream_install:new install tag added for rhel9

### PR DESCRIPTION
**Pull request by aliang123**
Add new install tag for blockdev_stream_install,
blockdev_snapshot_install, blockdev_mirror_install,
blockdev_commit_install

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2007553

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_snapshot_install.q35: PASS (954.74 s)
(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_install.q35: PASS (723.76 s)
(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_commit_install.q35: PASS (590.52 s)

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

**Comment by zhencliu** [comment]
LGTM

File: qemu/tests/cfg/blockdev_stream_install.cfg [hunk 1]
```diff
@@ -37,6 +37,7 @@
 
     tag_for_install_start = "Starting Login Service|Starting Update is Completed"
     tag_for_install_start += "|Mounted Temporary Directory"
+    tag_for_install_start += "|Anaconda NetworkManager configuration"
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_stream_install.cfg:40]: this tag is different from others, I guessed you may hit the issue that 'NetworkManager autoconnections configuration for Anaconda installation environment" was not found during installation, then what about adding both?

Comment (aliang123) [reply qemu/tests/cfg/blockdev_stream_install.cfg:40]: In fact, ppc don't support this tag, and the related tag is "NetworkManager auonda installation environment" in ppc, so, to make the same tag for both x86 and ppc, I re-set the tag to "Anaconda NetworkManager configuration" which are supported on both arch.

Comment (aliang123) [reply qemu/tests/cfg/blockdev_stream_install.cfg]: After discussion with ngu, updated patch according to zhenchao's suggestion.

Comment (zhencliu) [reply qemu/tests/cfg/blockdev_stream_install.cfg]: make sense, LGTM

**Comment by ngu-niny** [comment]
The failed case is for a product bug, others are PASS.

1-Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_mirror_install Finshed 2021-11-04 23:25:29 2021-11-04 23:30:39 ERROR 309.533548
2-Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_commit_install Finshed 2021-11-04 23:30:39 2021-11-04 23:50:02 PASS 1163.638528
3-Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_snapshot_install Finshed 2021-11-04 23:50:02 2021-11-05 00:09:46 PASS 1183.965407
4-Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_install Finshed 2021-11-05 00:09:47 2021-11-05 00:30:49 PASS 1262.037243

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**